utils.py

```diff
@@ -313,19 +350,42 @@ def update_article(article, issue, prepared_row, zip_folder_path):
     return article
 
 
+def update_keywords(keywords, article):
+    new_keywords = [w.strip(whitespace) for w in keywords]
```

Comment:
There are certan scenarios where an empty keyword might be introduced here, it is best to be safe and write this as: 
`new_keywords = [w.strip(whitespace) for w in keywords if w]`